devoxx-2017-be/cinema/src/main/java/org/tweetwallfx/devoxx2017be/dataprovider/SessionData.java

```diff
@@ -54,36 +73,25 @@
                 .map(entry -> entry.getValue().stream().findFirst())
                 .map(optional -> optional.orElse(null))
                 .map(slot -> new SessionData(slot.getRoomName(),
-                        slot.getTalk().getSpeakers().stream().map(ref -> ref.getName()).collect(Collectors.toList()),
```

Comment:
This indent change should be reverted. This allows for better reading of the parameter list of the constructor call.